thinc/layers/sparselinear.pyx

```diff
@@ -110,41 +127,57 @@ class _finish_linear_update:
 
 cdef void set_scoresC(float* scores,
         const uint64_t* keys, const float* values, const int32_t* lengths,
-        int batch_size, int nr_out,
-        const float* weights, int nr_weight) nogil:
+        int batch_size, int nr_out, const float* weights, int nr_weight,
+        bint invalid_indexing) nogil:
```

Comment:
I think it'd be a good idea to add a comment clarifying what the `invalid_indexing` parameter is used for (so that one wouldn't have to go through the Git history to understand why it was introduced).

Reply:
Done.